src/Microsoft.Kusto.ServiceLayer/LanguageServices/ConnectedBindingQueue.cs

```diff
@@ -157,24 +96,10 @@ public string AddConnectionContext(ConnectionInfo connInfo, bool needMetadata, s
                 try
                 {
                     bindingContext.BindingLock.Reset();
-                   
```

Comment:
We don't need ServerConnection so the below logic isn't needed. In addition, this line always threw an exception. `bindingContext.SmoMetadataProvider = SmoMetadataProvider.CreateConnectedProvider(bindingContext.ServerConnection);`